release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -1,17 +1,43 @@
 [id='con-relnotes-fixed-issues_{context}']
 = Fixed issues
 
-== Security issues resolved in {product} 1.1.0 and 1.1.1
+== Security issues resolved in {product} 1.1.z
```

Comment:
@Gerry-Forde revert to following the IBM guidance for [References to earlier or later versions](https://www.ibm.com/docs/en/ibm_style/pdf/ibm-style-documentation.pdf) for all instances of _.z_

Reply:
Thanks @linfraze